### DISCUSSION TITLE
How to change the color of the navigation drawer?

### DISCUSSION THREAD

**Discussion by maeries** [requarks/wiki#4947] [Help / Questions] [answered]
I'm a complete CSS noob, so bear with me. I want to change th color of the left blue sidebar. So I put 
```
.v-application .primary {
  background-color: #333739!important;
  border-color: #333739!important
}
```
into the custom CSS field. Why does that not work?
Firefox shows
![grafik](https://user-images.githubusercontent.com/30597531/151662855-22d286f3-f22b-4509-a7fb-dbb11a1d0950.png)
Why does the default values overwrite mine? Shouldn't it be the other way around?

**Comment by maeries** [comment]
Still stuck at this issue. Anyone got a hint for me?

**Comment by jamsondu** [comment, marked as answer]
<style type="text/css">
.v-application .primary{
  background-color: #333739!important;
  border-color: #333739!important
}
</style>


Copy the code to "Admin Area -> Theme -> Code Injection -> CSS Override -> Body HTML Injection"

The CSS selector of the same name takes the last loaded style.
